BrowserKit/Sources/ComponentLibrary/CardView.swift

```diff
@@ -1,87 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/
-
-import Common
-import UIKit
-
-/// The view model used to configure a CardView
-public struct CardViewModel {
```

Comment:
@thatswinnie we should keep this too right?

Reply:
@lmarceau not sure - it's not used anywhere. We could always bring it back in case we need it.